packages/flutter/lib/src/widgets/fade_in_image.dart

```diff
@@ -68,6 +68,9 @@ class FadeInImage extends StatefulWidget {
   /// The [placeholder] and [image] may be have their own BoxFit settings via [fit]
   /// and [placeholderFit].
   ///
+  /// The [placeholder] and [image] may be have their own FilterQuality settings via [filterQuality]
```

Comment:
nit: remove "be"?

Reply:
i was initially against it because the comment above had the same typo and i didn't want to include this inside the PR in favor of coherence.

Reply:
https://github.com/flutter/flutter/pull/110096/commits/7c6f40ddea0507cb5bee60def426e81fa2da935d fixes it.